cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -146,62 +196,125 @@ <h4>{{ email_link.label }}</h4>
+                Working at CFPB
+            </a>
+        </li>
+        <li class="m-list_item">
+            <a href="/office-civil-rights/" class="m-list_link">
+                Office of Civil Rights
+            </a>
+        </li>
+    </ul>
     <p>
-        Consider exploring our information about working at the CFPB
-        and our application process.
-    </p>
+       The Consumer Financial Protection Bureau (CFPB) is an equal
+       opportunity employer and seeks to create and maintain a vibrant
+       and diverse workforce. Women, minorities, veterans, and
+       people with disabilities are encouraged to apply.
+    </p>   
+</div>
+<div class="block block__flush-top">
+    <header class="m-slug-header">
+        <h2 class="a-heading">
+            Learn more about CFPB's work
+        </h2>
+    </header>
     <ul class="m-list m-list__links u-mb0">
         <li class="m-list_item">
-            <a href="/about-us/careers/working-at-cfpb" class="m-list_link">
-                Working at the CFPB
+            <a href="/about-us/blog" class="m-list_link">
+                CFPB Blog
             </a>
         </li>
         <li class="m-list_item">
-            <a href="/about-us/careers/application-process" class="m-list_link">
-                Learn how the application process works
+            <a href="/about-us/newsroom" class="m-list_link">
+                Newsroom
             </a>
         </li>
     </ul>
 </div>
-
 <div class="block block__flush-top">
-    {%- import 'related-links.html' as related_links -%}
-    {{- related_links.render([
-      [ '/about-us/blog/',
-        'CFPB Blog' ],
-      [ '/about-us/newsroom/',
-        'Newsroom' ],
-      [ '/office-civil-rights/',
-        'Office of Civil Rights' ]
-    ]) -}}
+    <header class="m-slug-header">
+        <h2 class="a-heading">
+            Follow us on LinkedIn
+        </h2>
+    </header>
+    <p data-qa-hook="info-section-desc">
+        The CFPB is one of the most searched for agencies
+        in the federal government.
+        Connect with us to stay updated on the work
+        we do and new opportunities to be a part of it.
+    </p>
+    <a href="https://www.linkedin.com/company/consumer-financial-protection-bureau"
+       class="a-link a-link__jump a-link__external-link">
+        Follow us on LinkedIn
+    </a>
 </div>
 
-<header class="m-slug-header">
-    <h2 class="a-heading">
-        Follow us on LinkedIn
-    </h2>
-</header>
-<p data-qa-hook="info-section-desc">
-    The CFPB is one of the most searched for agencies
-    in the federal government.
-    Connect with us to stay updated on the work
-    we do and new opportunities to be a part of it.
-</p>
-<a href="https://www.linkedin.com/company/consumer-financial-protection-bureau"
-   class="a-link a-link__jump a-link__external-link">
-    Follow us on LinkedIn
-</a>
+{% endblock %}
+
+{% block javascript %}
+{{ super() }} 
+
+<script type="application/ld+json">
```

Comment:
Oh interesting, I hadn't seen schema publishing in JS before. We had a schema discussion a long, long time ago but it never went anywhere. Should we look into this for other content as well?

Reply:
I think it would make sense to look into that further. There's probably other information (contacts, etc) that we could make more discoverable for search engines using structured data.